examples/chart/teleport-cluster/templates/auth/_config.aws.tpl

```diff
@@ -16,9 +20,20 @@
     write_min_capacity: {{ required "aws.writeMinCapacity is required when aws.dynamoAutoScaling is true" .Values.aws.writeMinCapacity }}
     write_max_capacity: {{ required "aws.writeMaxCapacity is required when aws.dynamoAutoScaling is true" .Values.aws.writeMaxCapacity }}
     write_target_value: {{ required "aws.writeTargetValue is required when aws.dynamoAutoScaling is true" .Values.aws.writeTargetValue }}
-    {{- else }}
+  {{- else }}
     auto_scaling: false
+  {{- end }}
+  {{- if .Values.aws.accessMonitoring.enabled }}
```

Comment:
can you format this line? It's difficult to understand that the code bellow belongs to this if block

Reply:
but the indentation reflects the gotemplate block level 🤔